.github/workflows/certification.yml

```diff
@@ -245,10 +252,65 @@ jobs:
           exit 1
         fi
 
+    - name: Prepare Cert Coverage Info
+      run: |
+       mkdir -p path/to/cov_files
+       SOURCE_PATH_LINEAR=$(echo ${{ env.SOURCE_PATH }} |sed 's#/#\.#g')
+       echo "${{ env.cover_per_only }}" >> path/to/cov_files/$SOURCE_PATH_LINEAR.txt
+
+    - name: Upload Cert Coverage Info
+      uses: actions/upload-artifact@v3
+      with:
+        name: cert_cov
+        path: path/to/cov_files
+        retention-days: 1
+
+    - name: Discord notification
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_MONITORING_WEBHOOK_URL }}
+      uses: Ilshidur/action-discord@0c4b27844ba47cb1c7bee539c8eead5284ce9fa9
+      with:
+        args: 'Cert Test Coverage for {{ SOURCE_PATH }} is {{ cover_per_only }}'
+
     # Upload logs for test analytics to consume
     - name: Upload test results
       if: always()
       uses: actions/upload-artifact@master
       with:
         name: ${{ matrix.component }}_certification_test
         path: ${{ env.TEST_OUTPUT_FILE_PREFIX }}_certification.*
+
+  post_job:
+    name: Notify Total coverage
+    runs-on: ubuntu-latest
+    needs: certification
+    if: always()
```

Comment:
Is this needed?

Reply:
Yes, it is needed - without `needs`, it will not run post all jobs in certification, rather can run before that.
Without `if: always()`, it can be skipped.
Is there any other part you are referring?

Reply:
The comment was for `if: always()`.